ci/rpm_build_success.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# Script to be run on successful RPM build
+
+mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+ci_envs="$mydir/ci/parse_ci_envs.sh"
+if [ -e "${ci_envs}" ]; then
+  # shellcheck disable=SC1091
+  source "${ci_envs}"
```

Comment:
(lint) In POSIX sh, 'source' in place of '.' is not supported. [SC2039]

Reply:
Shouldn't we be abiding by this one?

Reply:
That was fixed 16 days ago.